logging-backend/README.md

```diff
@@ -0,0 +1,51 @@
+## Spine Logging Backend for JVM
+
+This module defines basic mechanisms that handle log records produced by 
+the logging facade.
+
+### Default Platform
+
+`DefaultPlatform` is mostly responsible for runtime discovery of injectable 
+services and providing log site determination mechanism. It uses `java.util.ServiceLoader`
+to find available service implementations, and stack trace analysis to determine
+a log site.
+
+The following services may be injected:
+
+| Service               | Default |
+|-----------------------|---|
```

Comment:
Let's format the table borders to fit the right edge here and in the following lines.

Reply:
IDEA Markdown plugin can do it automatically.